style.css

```diff
@@ -0,0 +1,79 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-size: 16px;
+}
+
+/* the header section */
+header {
+  background: #3c3a39;
+  display: flex;
+  justify-content: space-between;
+  color: #fff;
+  padding: 15px;
+}
+
+.logo {
+  font-family: 'Roboto', sans-serif;
+  line-height: 24px;
+  font-weight: 700;
+  font-size: 18px;
+}
+
+#logo {
+  color: #fff;
+  text-decoration: none;
+}
+
+/* headline section */
+
+.headline {
+  background: #1c1a19;
+  color: #fff;
+  padding: 130px 25px 145px;
+  background-image: url(./Images/Group.png);
+  background-position: bottom right;
+  background-repeat: no-repeat;
+}
```

Comment:
- [ ] Dear @arnoldnekemiah, It's among the primary requirement to use flex-box to align contents inside this section. This gives us a lot of options to align all our elements, plus, it's responsive.